app/controllers/api/v1/concerns/authenticator.rb

```diff
@@ -46,7 +46,8 @@ def current_api_user
                 error_context = {
                     auth_response_body: response.present? ? response.body : nil,
                     auth_response_code: response.present? ? response.code : nil,
-                    auth_response_headers: response.present? ? response.headers : nil
+                    auth_response_headers: response.present? ? response.headers : nil,
+                    api_access_token: api_access_token
                 }
```

Comment:
do we really want to log the token -- isn't that a security issue?  Should we maybe log some subset of the token or metainfo e.g. whether it's nil/malformed or somesuch?

Reply:
If it were a refresh token, I would agree - but OAuth tokens only last for an hour, so I think the issue here is negligible.  The main issue I'm trying to solve here is that when there's an error without a response, we don't know anything else about the request.  The token would be the only other piece of information we have.

Having said that, I think your approach of just logging that there _was_ a token would help - if we see errors saying there was no token.  Also, after some reading and testing, it does appear that this API endpoint will respond `400` if the token has expired, rather than `401`.  It looks like this updated endpoint behaves more normally: https://www.googleapis.com/oauth2/v3/userinfo.

I'm going to make some more substantive updates so that we have better behavior on this method for expired auth tokens.

Reply:
ok, then lets do that of just logging presence/absence (and maybe last 5 chars if it's present or somesuch).  I can see FISMA auditors not liking seeing tokens in our logs, even if we tell them that they were probably expired

Reply:
After some more reading/testing, the above endpoint works, but doesn't give us any info about the _token_, just the _user_.  More reading has shown that the `400` on expired tokens for the token_info endpoints is expected behavior.  So we may just want to catch the `400` explicitly and treat it like a 401.

Reply:
346dbd9